Sources/Opus/Opus.Decoder.swift

```diff
@@ -22,6 +23,27 @@ public extension Opus {
 			}
 		}
 
+		public init(customOpus: OpaquePointer,
```

Comment:
Given the encoder and decoder typically exist on different machines, I think `Decoder` should own the `OpusCustomMode` and create it here.

It can be freed in `deinit`.

Reply:
I disagree - custom encoders and decoders need the custom object; you need to have a custom object in order to use either.
It maybe in your use case you have encoding on one machine and decoding on another, but you have to have the same custom instance for both if you're using them together.

Reply:
The constructor for an `OpusCustomMode` takes two arguments: a sample rate and a target frame size in number of samples, which must be 64-1024 plus some additional constraints on prime factors.

The sample rate can be derived from the `AVAudioFormat` in the constructor along with a `customFrameSize` parameter to the custom constructor for an `Encoder` and `Decoder`.

It’s probably useful to have common helper code to aid in the creation of a valid `OpusCustomMode` that throws an error, but isn’t part of this package’s public API.

Reply:
> It’s probably useful to have common helper code to aid in the creation of a valid `OpusCustomMode` that throws an error, but isn’t part of this package’s public API.

That is the Opus.Custom.swift file.